doc/rst/usingchapel/prereqs.rst

```diff
@@ -40,8 +40,8 @@ about your environment for using Chapel:
   * If you wish to use chpldoc or Chapel's test system, ``curl`` and
     python-devel (or equivalent packages for your platform) are required.
 
-  * If you wish to use the ``mason system``, chapel's package manager, ``pkg-config`` is
-    required.
+  * If you wish to use the ``mason system``, chapel's package manager, ``pkg-config`` and
+    ``git`` are required.
```

Comment:
Is `git` not required for typical `mason` usage? For example, `mason new` creates a new git repo, and packages are pulled via `git`.

Reply:
Because we added the options for offline usage such as `--no-init` for `mason new` and `--no-update-registry` for `mason update` and `mason build`, technically `git` is not required for building libs/apps without dependencies. 

I might mention a line that says something about the functionality of mason being significantly limited without `git`. 

We might consider making an `MASON_OFFLINE` mode where those arguments are added automatically.

Reply:
I see. I suspect there may be some cases where we would be using `git` even in offline mode, e.g. checking out a different version of a pre-installed library.

For now, it may be safer to state Mason depends on `git` until we have an offline "mode" available via environment variable, and have sufficiently tested `git`-less mason usage.

Reply:
`git` dependency aside, I'd suggest rewording this sentence:

```
* If you wish to use :ref:`readme-mason`, chapel's package manager,  ``git`` is required.
    - The ``mason system`` subcommands additionally require ``pkg-config`` 
```